### ISSUE TITLE
Missing require in closure library for goog.a11y.aria.datatables

### ISSUE THREAD

**Issue by gramic** [bazelbuild/rules_closure#550]
In the [datatables.js](https://github.com/google/closure-library/blob/d31065ce7d26bbdedc4a364578fbb106d847f9da/closure/goog/a11y/aria/datatables.js#L28) file, there is this line:
```js
goog.a11y.aria.DefaultStateValueMap_;
```
The `goog.a11y.aria` is not required by `goog.require`. Recently in the history of the file, there was a `@suppress` for missing require, but was removed at the current version.
I forked the library, just to add a `@suppress {missingRequire}` again, but maybe there is another way to do this without modifying `closure-library` source code?

On a side note: Thank you very much for including bazel BUILD files inside the closure library!

**Comment by gkdn** [comment]
@kjin Do you know why the suppression why the suppression was removed? I saw the CL but there wasn't any clarification why it was no longer needed.

**Comment by gkdn** [comment]
I have just updated to the latest versions of Closure Compiler and Library to take that out of the equation. @gramic could you check if the problem still exits?

**Comment by gramic** [comment]
I forgot to mention, that I did rule that out before with `omit_com_google_javascript_closure_library = True,` with the latest compiler/library and the error still exists.  Anyway, just tried with your updates and there is no difference.

**Comment by gkdn** [comment]
@shicks @kjin Any ideas here?

**Comment by shicks** [comment]
Kelvin is no longer at Google.

@concavelenz cleaned up the `missingRequire` as part of a global cleanup he did a few quarters ago.  I don't know if he can give any additional context.  It was added by @nreid260 (who is also no longer on the JS teams), so the removal CL was essentially just a rollback of the CL that added it.  I have no idea why it was added in the first place (from reading the linked bug, it seems like there was a compiler check that was made stricter, but it's puzzling that it was removable without actually changing anything in the file).

Quoting from the bug:
> In particular, references to this bug were inserted in code where the autofixer was unable to fix the code automatically.
>
> The most common cause of this is when we either could not figure out which module to import (e.g. your code references foo.bar but there are multiple definitions of foo.bar in the codebase), or we figured out which module but we couldn't figure out how to modify your BUILD file to depend on it (due to macros).

What problem is this causing you?  As far as I can tell, this isn't causing problems for anybody internally, so maybe there's some different configuration somewhere in the bazel versions?

Looking more at the underlying issue, it seems like the obvious fix is to move the problematic symbol into the provided namespace?  Can you give that a try and see if it solves the problem without re-adding the suppression?  (i.e. rename `goog.a11y.aria.DefaultStateValueMap_` to `goog.a11y.aria.datatables.DefaultStateValueMap_`)  That's probably what it should have been anyway.  I'm currently working on making that change internally, but I'd like confirmation that it solves your problem before I submit it.

**Comment by gramic** [comment]
Doing it exactly as you suggested, works without errors. Here is the [patch](https://github.com/gramic/closure-library/commit/2085f647dd677503703d4a2e1267d1eebf3e9e43) that shows it.
